src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/default-transcripts-list-item/DefaultTranscriptListItem.tsx

```diff
@@ -57,10 +67,34 @@ export const DefaultTranscriptListItem = (
     cursor: 'pointer'
   };
 
+  const defaultTranscriptLabelMap = {
+    select: {
+      label: 'Selected',
+      helpText:
+        'The selected transcript is a default single transcript per protein coding gene that is representative of biology, well-supported, expressed and highly conserved'
+    },
+    plus: {
+      label: 'MANE Plus',
+      helpText:
+        'The selected transcript is a default single transcript per protein coding gene that is representative of biology, well-supported, expressed and highly conserved'
+    }
+  };
+
+  const isCanonical = props.isDefault || false;
+  const canonicalType = 'select'; // TODO Change this to transcript.mane/plus etc when available
```

Comment:
From code review:

1) We agreed that isCanonical is misleading, because only one transcript among all gene transcripts will be canonical, but several transcripts can have a label in the left-hand side of the component
2) We agreed not to try to predict the future, and to remove the reference to MANE Plus from this component. We will have to revisit this component anyway when we start getting real data from the backend.